src/Components/v2/CarouselV3.tsx

```diff
@@ -1,20 +1,25 @@
 import { ChevronIcon, color, Flex, space } from "@artsy/palette"
+import { Options as FlickityOptions } from "flickity"
 import React, { Fragment } from "react"
-import { FlickityOptions } from "react-flickity-component"
 import styled from "styled-components"
 import { left, LeftProps, right, RightProps } from "styled-system"
 import { Media } from "Utils/Responsive"
 
```

Comment:
@damassi Happy to change this file to CarouselV4 before we merge but replaced CarouselV3 so you can get better diffs. Everything has been tested and the changes aren't major so we may not need a V4, let me know which you prefer.